db/system_keyspace.cc

```diff
@@ -1646,6 +1661,14 @@ future<std::optional<db_clock::time_point>> get_saved_cdc_streams_timestamp() {
     });
 }
 
+future<bool> cdc_is_rewritten() {
+    // We don't care about the actual timestamp; it's additional information for debugging purposes.
+    return qctx->execute_cql(format("SELECT key FROM system.{} WHERE key = ?", v3::CDC_LOCAL), "rewritten")
+            .then([] (::shared_ptr<cql3::untyped_result_set> msg) {
+        return !msg->empty() && msg->one().has("key");
```

Comment:
Is it possible that !msg->empty() and !msg->one().has("key")?

Reply:
No

Reply:
Can we drop the msg->one().has("key") check then?

Reply:
Yes, if you really want